workspaces/acs/packages/app/package.json

```diff
@@ -0,0 +1,81 @@
+{
+  "name": "app",
+  "version": "0.0.0",
+  "private": true,
+  "bundled": true,
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/backstage/community-plugins",
+    "directory": "workspaces/acs/packages/app"
+  },
+  "backstage": {
+    "role": "frontend"
+  },
+  "scripts": {
+    "start": "backstage-cli package start",
+    "build": "backstage-cli package build",
+    "clean": "backstage-cli package clean",
+    "test": "backstage-cli package test",
+    "lint": "backstage-cli package lint"
+  },
+  "dependencies": {
+    "@backstage-community/plugin-acs": "^0.1.0",
```

Comment:
```suggestion
    "@backstage-community/plugin-acs": "workspace:^",
```

The should fix the build failure. That tells yarn to look for this package in the current workspace.